api/android/build-android-lib.sh

```diff
@@ -16,6 +16,32 @@ nnstreamer_target_abi="'armeabi-v7a', 'arm64-v8a'"
 # Set tensorflow-lite version (available: 1.9 and 1.13)
 nnstreamer_tf_lite_ver=1.13
 
+# Variables to release library (GROUP:ARTIFACT:VERSION)
+release_bintray='no'
+release_group='com.nnsuite'
```

Comment:
As we've voted today, let's make our package name as "org.nnsuite.nnstreamer".

Reply:
Group : org.nnsuite
Artifact : nnstreamer

I will update the package name of API and sample in next week.